src/littlefs/context.py

```diff
@@ -74,3 +74,89 @@ def sync(self, cfg: 'LFSConfig') -> int:
             Filesystem configuration object
         """
         return 0
+
+
+
+# if the user has win32file installed then define this class
+try:
```

Comment:
try to avoid large try/excepts, as you could catch unintended exceptions. I would:

```
import ctypes
try:
    import win32file
except ImportError:
    pass
else:
    # Put the UserContextWinDisk implementation here.
```

Reply:
This is what I want to do! Didn't know I could do that. Also, instead of `pass`, I think it might be better to add a message to warn the user.